datahub-kubernetes/datahub/charts/datahub-frontend/templates/deployment.yaml

```diff
@@ -84,10 +90,26 @@ spec:
               value: "{{ .Values.datahub.play.mem.buffer.size }}"
             - name: KAFKA_BOOTSTRAP_SERVER
               value: "{{ .Values.global.kafka.bootstrap.server }}"
+            {{- if .Values.global.springKafkaConfigurationOverrides }}
+            {{- range $configName, $configValue := .Values.global.springKafkaConfigurationOverrides }}
+            - name: KAFKA_PROPERTIES_{{ $configName | replace "." "_" | upper }}
+              value: {{ $configValue }}
+            {{- end }}
+            {{- end }}
+            {{- if .Values.global.credentialsAndCertsSecrets }}
+            {{- range $envVarName, $envVarValue := .Values.global.credentialsAndCertsSecrets.secureEnv }}
+            - name: KAFKA_PROPERTIES_{{ $envVarName | replace "." "_" | upper }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $.Values.global.credentialsAndCertsSecrets.name }}
+                  key: {{ $envVarValue }}
+            {{- end }}
+            {{- end }}
             - name: ELASTIC_CLIENT_HOST
               value: "{{ .Values.global.elasticsearch.host }}"
             - name: ELASTIC_CLIENT_PORT
               value: "{{ .Values.global.elasticsearch.port }}"
+            {{- if .Values.global.datahub_analytics_enabled }}
```

Comment:
Why is this located here as opposed to above?

Reply:
Good point. Done